src/oc/hoc.cpp

```diff
@@ -1639,11 +1632,7 @@ int hoc_yyparse(void) {
 		i = yyparse();
 		hoc_in_yyparse = 0;
 		switch (i) {
-		case 'e':
-			i = '\n';
-			hoc_edit();
-			break;
-		case -3 : /* need more input */
+        case -3 : /* need more input */
```

Comment:
why was the `'e'` switch case removed?

Reply:
by mistake, nice catch !

Reply:
Removal of this code was breaking microemacs. Now microemacs users are safe :)